src/modules/beamos/config

```diff
@@ -6,17 +6,21 @@
 TRUE_USER=pi
 
 # Branches
-[ -n "$BEAMOS_MRBEAMPLUGIN_REPO_BRANCH" ] || BEAMOS_MRBEAMPLUGIN_REPO_BRANCH=mrbeam2-stable
+## Plugins
+[ -n "$BEAMOS_MRBEAMPLUGIN_REPO_BRANCH" ] || BEAMOS_MRBEAMPLUGIN_REPO_BRANCH=v0.11.0
 [ -n "$BEAMOS_CAMERA_REPO_BRANCH" ] || BEAMOS_CAMERA_REPO_BRANCH=mrbeam2-stable
-[ -n "$BEAMOS_MRBEAMIOBEAM_REPO_BRANCH" ] || BEAMOS_IOBEAM_REPO_BRANCH=mrbeam2-stable
-[ -n "$BEAMOS_MRBEAMLED_REPO_BRANCH" ] || BEAMOS_MRBEAMLED_REPO_BRANCH=mrbeam2-stable
+[ -n "$BEAMOS_FINDMYMRBEAM_REPO_BRANCH" ] || BEAMOS_FINDMYMRBEAM_REPO_BRANCH=v0.4.0
+[ -n "$BEAMOS_NETCONNECTD_PLUGIN_REPO_BRANCH" ] || BEAMOS_NETCONNECTD_PLUGIN_REPO_BRANCH=v0.2.0
```

Comment:
As discussed we both don't like the naming since sometimes is a version and sometimes a branch but we will tackle it as part of the following ticket ✅